src/commands/skip.rs

```diff
@@ -0,0 +1,66 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use serenity::builder::CreateApplicationCommand;
+use serenity::{async_trait, model::prelude::command::CommandOptionType::Integer};
+
+use crate::command_context::CommandContext;
+use crate::store::guild_action::play_next_track::PlayNextTrack;
+use crate::store::guild_action::remove::Remove;
+
+use super::{CommandDataOptionUtil, JoovyCommand, JoovyCommands};
+
+const FROM: &str = "from";
+const TO: &str = "to";
+
+#[derive(Default, Debug)]
+pub struct Skip;
+
+#[async_trait]
+impl JoovyCommand for Skip {
+    fn create_application_command<'a>(
+        &self,
+        command: &'a mut CreateApplicationCommand,
+    ) -> &'a mut CreateApplicationCommand {
+        command
+            .name(JoovyCommands::Skip(Skip).as_ref())
+            .description("Removes tracks from the queue. If from is not defined the current track will be skipped.")
```

Comment:
"... If no arguments are given, the current track will be skipped."

Reply:
Är bättre. Fixar!